src/main/java/com/ibm/devops/connect/ReconnectExecutor.java

```diff
@@ -36,23 +41,39 @@ public void startReconnectExecutor() {
     }
 
     private class ReconnectRunner implements Runnable {
-
         @Override
         public void run()
         {
             try {
-                if (!cloudSocketInstance.isAMQPConnected() && Jenkins.getInstance().getDescriptorByType(DevOpsGlobalConfiguration.class).isConfigured()) {
-                    try {
-                        log.info("Reconnecting to AMQP");
-                        cloudSocketInstance.connectToAMQP();
-                    } catch (Exception e) {
-                        log.error("Unable to Reconnect to UCV AMQP", e);
+                if (checkIfConfigured()) {
+                    if (!cloudSocketInstance.isAMQPConnected()) {
+                        try {
+                            log.info("Reconnecting to AMQP");
+                            cloudSocketInstance.connectToAMQP();
+                        } catch (Exception e) {
+                            log.error("Unable to Reconnect to UCV AMQP", e);
+                        }
                     }
                 }
             }
             finally {
```

Comment:
empty finally block? if there is nothing in it, probably remove it and its try statement.

Reply:
removed